src/Altinn.Platform/Altinn.Platform.Authentication/Altinn.Platform.Authentication.Tests/Controllers/AuthenticationControllerTests.cs

```diff
@@ -761,6 +763,8 @@ public async Task AuthenticateUserWithOIDCExternalIDentity_FullProcess_Redirects
             Assert.NotNull(claimPrincipal);
             Assert.NotNull(claimPrincipal.Claims.FirstOrDefault(r => r.Type.Equals("urn:altinn:userid")));
             Assert.Equal("234234",  claimPrincipal.Claims.FirstOrDefault(r => r.Type.Equals("urn:altinn:userid")).Value);
+            Assert.NotNull(claimPrincipal.Claims.FirstOrDefault(r => r.Type.Equals("urn:feide:role")));
+            Assert.Equal(2, claimPrincipal.Claims.Where(r => r.Type.Equals("urn:feide:role")).Count());
```

Comment:
It's possible to simplify the query slightly. Sommething like this:
```suggestion
            Assert.Equal(2, claimPrincipal.Claims.Count(r => r.Type.Equals("urn:feide:role")));
```